docs/acm/component/server-foundation/metrics.md

```diff
@@ -8,10 +8,11 @@ Nomenclature:
 | acm_managed_cluster_count  | - | Managed cluster count | None |  Not exposed, STABLE |
 | acm_managed_cluster_addon_status_condition | gauge | Managed cluster addon status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`addon_name`</td> <td>The name of the add-on.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_id`</td> <td>The name of the managed cluster to which the add-on belongs.</td> </tr> <tr> <td>3</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster to which the add-on belongs.</td> </tr> <tr> <td>4</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>5</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions, such as `Available`, the status is set to unknown if the condition is missing. </td> </tr></table> | Not exposed |
-| acm_managed_cluster_status_condition | gauge | Managed cluster status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an {ocp-short} cluster, otherwise it is the cluster name.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster.</td> </tr> <tr> <td>3</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>4</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions, such as `ManagedClusterConditionAvailable`, the status is set to unknown if the condition is missing. </td> </tr></table>  | Not exposed |
+| acm_managed_cluster_status_condition | gauge | Managed cluster status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an {ocp-short} cluster. Otherwise, managed cluster ID is the cluster name.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster.</td> </tr> <tr> <td>3</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>4</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions, such as `ManagedClusterConditionAvailable`, the status is set to unknown if the condition is missing. </td> </tr></table>  | Not exposed |
```

Comment:
```suggestion
| acm_managed_cluster_status_condition | gauge | Managed cluster status condition | <table> <tr> <th>Number</th> <th>Label</th> <th>Description</th> </tr> <tr> <td>1</td> <td>`managed_cluster_id`</td> <td>The managed cluster ID is the cluster ID if the managed cluster is an {ocp-short} cluster. Otherwise, the managed cluster ID is the cluster name.</td> </tr> <tr> <td>2</td> <td>`managed_cluster_name`</td> <td>The name of the managed cluster.</td> </tr> <tr> <td>3</td> <td>`condition`</td> <td>The name of the condition.</td> </tr> <tr> <td>4</td> <td>`status`</td> <td>The status of the condition. Valid values are `true`, `false`, or `unknown`. For required conditions, such as `ManagedClusterConditionAvailable`, the status is set to unknown if the condition is missing. </td> </tr></table>  | Not exposed |
```